src/client/components/sidebar/sidebar.js

```diff
@@ -42,13 +49,13 @@ class Sidebar extends React.Component {
           </Button>
           <hr />
           <Leaderboard playerID={this.props.playerID} scores={this.props.G.scores} names={this.props.names} cards={getDealtCardsForPlayers(this.props.G.order, this.props.G.dealt)} />
-          <hr />
-          <Button color="secondary" size="lg" block disabled={
+          <p className={(isLastToPass) ? "visible" : "invisible"} >You are the last one to pass!</p>
+          <Button color={(isLastToPass) ? "warning" : "secondary"} size="lg" block disabled={
               this.props.ctx.phase !== "threats" ||
               this.props.G.passed.includes(this.props.playerID) ||
               !this.props.active
             } onClick={() => { this.props.moves.pass() }}>
-              Pass
+              PASS
```

Comment:
I think we should go back to `Pass` as this is how buttons are written everywhere else